externs/react.ext.js

```diff
@@ -0,0 +1,76 @@
+/** @suppress {duplicate} */
```

Comment:
While there is some value in having a clearly defined API somewhere (which can also be used as an externs file for other project, even better if it had type annotations), why are you not using the `@expose` annotation instead?